src/integrationTest/java/org/opensearch/security/privileges/PrivilegesEvaluatorTest.java

```diff
@@ -68,4 +73,11 @@ public void testRegexPattern() throws Exception {
         }
 
     }
+
+    @Test
+    public void testSearchTemplateRequest() {
```

Comment:
Before adding search template into isClusterPerm function this would fail since it would come back with a 403 response code